app/models/notification.rb

```diff
@@ -79,7 +83,22 @@ def unarchive_if_updated
   def update_from_api_response(api_response, unarchive: false)
     attrs = Notification.attributes_from_api_response(api_response)
     self.attributes = attrs
+    set_author(api_response)
     unarchive_if_updated if unarchive
     save(touch: false) if changed?
   end
+
+  private
+
+  def set_author(api_response)
+    /\/((?:issues|pulls)\/(?<issue_number>\d+))|((?:commits)\/(?<commit_sha>[0-9a-f]{5,40}))\z/ =~ api_response[:subject][:url]
+    repo_name = api_response[:repository][:full_name]
+    self.subject_author = if issue_number
+                            issue = user.github_client.issue(repo_name, issue_number)
```

Comment:
Since the default Octobox doesn't ask for `repo` scope this will raise an `Octokit::NotFound` error when we try grab an issue for a repo we don't have access too.

Just wanted to point it out 😄